### ISSUE TITLE
'TypeError: computed.default.alias is not a function'

### ISSUE THREAD

**Issue by AW-UC** [miguelcobain/ember-cli-selectize#98]
Ember-CLI 1.13.8 & Ember-CLI-Selectize 0.4.0:
https://github.com/miguelcobain/ember-cli-selectize/blob/master/addon/components/ember-selectize.js#L21 is compiled to `placeholder: computed['default'].alias('prompt')` and throws: `Uncaught TypeError: computed.default.alias is not a function`.
This is an easy fix: Just use `Ember.computed` instead of `ember-new-computed`. Using the later only makes sense in combination with `get` or `set` but not with `alias`. If you want, I can submit PR. Just tell me if you want to keep using `ember-new-computed` at all or if it should be removed.

**Comment by AW-UC** [comment]
Hmm... just saw that it says *All `Ember.computed` helpers exists as well* in https://github.com/rwjblue/ember-new-computed/blob/master/README.md. So my above statement about when to use `ember-new-computed` is wrong and `computed.alias` shouldn't be a problem. Might be an `ember-new-computed` issue then.

**Comment by gte451f** [comment]
I'm seeing this same problem with the latest Ember CLI release (1.13.8).

**Comment by martinmaillard** [comment]
Same here. Is there a work-around ?

**Comment by AW-UC** [comment]
@martinmaillard Sure! If you use a recent Ember CLI, just avoid using `ember-new-computed`. If you depend on `ember-new-computed` because of using in an older Ember CLI, I'd recommend to let `ember-new-computed` only handle `get` & `set`. In both cases you will need to change `ember-cli-selectize` sources accordingly.

**Comment by martinmaillard** [comment]
I meant a work around in my app, without modifying ember-cli-selectize. I ended up just not upgrading to 0.4.0

**Comment by AW-UC** [comment]
Yes, there is no workaround like that. @miguelcobain You removed `ember-new-computed` in https://github.com/miguelcobain/ember-cli-selectize/commit/42c5f15309a06f3a34123fbe599eb02604233e22 than added it back via https://github.com/miguelcobain/ember-cli-selectize/commit/30ce9894b2f5259972d916faf2e47a6d97e08b6e. Was that on purpose?
